conda/base/context.py

```diff
@@ -531,6 +532,9 @@ def get_help_dict():
             this setting, specifying that certain files should never be soft-linked (see the
             no_link option in the build recipe documentation).
             """),
+        'alt_features': dals("""
+            Prioritize package version and build number maximization over feature optimization.
+            """),  # TODO: Use a more expressive name? Rephrase help text to be more user-friendly.
```

Comment:
A couple comments here at the design level of the config parameter.  We should *definitely* have a more descriptive name.  These user interface things are some of THE HARDEST things to change once they're live if we someday don't like it.

Also, I think it's best--for the time being--to

1. leave this configuration gate undocumented, which means removing the `alt_features` entry here, and instead adding the key to the list around line 469 (feel free to leave the description as a comment there), and
2. NOT add the command-line switch right now.  Because this is something of a beta feature right now, I don't think any of us are eager to expose it to the world, and support it indefinitely.  We still don't even know what it exactly means yet.  But as soon as we document it in `conda config --describe` and make a CLI flag for it, we're pretty much stuck with it.

So, with no CLI flag, how do we enable it for us power users here?  Just use it as an environment variable.

    CONDA_ALT_FEATURES=true conda install blarg

Or of course you can also stick it in a .condarc file somewhere.

Reply:
I agree on all the above. Who wants to come up with a better name?!
Does the configuration parameter suffice or do you think we need to add an environment variable?

Reply:
I guess I'd be nice to have it available on a per command/install basis, e.g., like in your example
```bash
CONDA_ALT_FEATURES=true conda install blarg
```
Will have to see where to plug this into to the code.
What are your thoughts about using a hidden CLI argument, i.e., something that uses `help=argparse.SUPPRESS`?

Reply:
A configuration parameter declared as `alt_features` will pick up any environment variable named `CONDA_ALT_FEATURES`.  So it's already taken care of.  Similarly, naming the argparse flag `--alt-features`, which turns into `args.alt_features`, will also be available as `context.alt_features`.  That is, all available configuration sources are merged into `context`, in the order `SEARCH_PATH`, environment variables, argparse args.

I don't know what a better name is yet either.  But, if it's undocumented right now, then I don't care quite as much.

Reply:
Uhh, I like you black magic 😄

Reply:
Yeah. This black magic has worked REALLY well though, since conda 4.2. You should read this: https://www.anaconda.com/blog/developer-blog/conda-configuration-engine-power-users/

Reply:
Really the whole `conda/common/configuration.py` was written to be extracted into its own library at some point. It’s had enough production use with conda now that it’s probably time to do some final cleanup and set it loose. If we only had the time...

Reply:
What about `--force-latest-version`?